en/api/cciapi.rst

```diff
@@ -905,6 +908,9 @@ cci_connect_with_url
         --connection URL string when useSSL property specified for encrypted connection
         URL = "cci:cubrid:192.168.0.1:33000:demodb:::?useSSL=true
 
+        --connection URL string when oracleStyleNumberReturn property is specified for remove decimal zeros from the result value
```

Comment:
Please refer to kisoo's comment below and modify it.

Reply:
thanks for the comment I'll fix it.